tests/test_global_buffer/verilator/test_io_address_generator.cpp.bak

```diff
@@ -0,0 +1,381 @@
+/*==============================================================================
```

Comment:
What's this file? Why named as `.bak`? If it's a backup file, please remove it.

Reply:
Ah. Removed this backup file.